vavr/src/test/java/io/vavr/collection/CharSeqTest.java

```diff
@@ -4017,4 +4017,9 @@ public void shouldConvertToShortUsingRadix() {
     public void shouldNotConvertToShortUsingRadixGivenEmptyCharSeq() {
         CharSeq.empty().toShort(2);
     }
+
```

Comment:
Please create a new comment in line 126:

```java
// -- compareTo(Object)
```

and move the `Comparable` tests there.